account_due_list/readme/CONFIGURE.rst

```diff
@@ -1,3 +1,4 @@
-To use this module, you need to :
+To use this module, you need to go to:
```

Comment:
Please put several lines with the steps, instead all of them together. We use `#. ` for auto-numbering them.

Reply:
@pedrobaeza & @victoralmau please review this PR.

TT

@Tecnativa